app/src/components/Pages/TipsAndArticles/Article.tsx

```diff
@@ -8,21 +8,23 @@ export type ArticleItem = {
   author: string;
   title: string;
   img: string;
+  content: string;
 }
 
 type ArticleProps = {
   text: string
-  url: string
+  link: string
   author: string
   img: string
+  content: string
```

Comment:
In the component of "add article" you remove the option for content input. So why do we need this prop in the article type?